src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -233,6 +233,16 @@
 public class ObjectInputStream
     extends InputStream implements ObjectInput, ObjectStreamConstants
 {
+    /**
+     * enable fast serialization:
```

Comment:
Does adding these field and code break our JCK compatibility?

Reply:
Adding a private attribute will not break TCK compatibility